source/qml/Library/RendererQml/AdaptiveCardQmlRenderer.cpp

```diff
@@ -535,7 +555,9 @@ namespace RendererQml
         backgroundTag->Property("color", choiceset.backgroundColor);
         backgroundTag->Property("border.color", "'grey'");
         backgroundTag->Property("border.width", "1");
+		backgroundTag->Property("implicitHeight", "35"); //TODO: To be replaced from hostconfig value 
```

Comment:
this should be `anchors.fill: parent`

Reply:
This happens when I used anchors.parent

![image](https://user-images.githubusercontent.com/78950710/110916725-1c8b4680-833f-11eb-8492-842a2fd81529.png)

The rectangle gets squished